src/components/pages/download/DownloadAccordion.astro

```diff
@@ -67,6 +67,23 @@ let downloadOptions = (await getCollection("download")).filter(
   const menuContainer = acc.querySelector(".download-accordion-menu");
 
   let activePanel = panels[0];
+
+  if (id === "download-nix-accordion") {
+    const userPlatform = navigator.userAgent.toLowerCase();
+    let os = "linux";
+    if (userPlatform.includes("macintosh")) {
+      os = "macos";
+    } else if(userPlatform.includes("windows")) {
+      os = "windows";
+    }
+    for (let i = 0; i < panels.length; i++) {
+      if (panels[i].id === `nix-install-${os}`) {
+        activePanel = panels[i];
+        break;
+      }
+    }
```

Comment:
```suggestion
    activePanel = acc.querySelector(`#nix-install-${os}`);
```

Reply:
That's clever. Actually I tried the same approach the first time but I used `panels` to query on but obviously that didnt work and somehow I overlooked the `acc` variable.